### ISSUE TITLE
Let me turn off GDevelop automatic updates.

### ISSUE THREAD

**Issue by ddabrahim** [4ian/GD#571]
Please stop updating GDevelop automatically.
Instead, prompt a message saying a new update is available and let me decide if I want to download it and install it.
Or, add an option to let me choose if I want GDevelop to update automatically or not.

It would be also important for security. If you release something broken that is going to corrupt my project file let me decide if I do want to take that risk or not. Project file can be corrupted also if GD crash while in use and write the project file since you are developing GDevelop on a Mac that only 1% of people using, there is no way for you to be sure everything is ok on Windows. Also in case your GtHub get hacked, currently it would be very simple to infect all GD5 users with virus and malware. Since you also take subscriptions in-app, I guess you don't want peoples card details being compromised as the direct result of this unsafe approach of updating GDevelop.

The bottom line is, for security reasons let me decide what do I want download and run.

Also, you did mentioned something about collecting crash reports and usage data automatically. If you have that in, let me turn that off too for the very same reasons. It can be exploited by viruses and malware and I don't care about how secure it is, the bottom line is, let me take full control what GDevelop doing on my computer.
.
Thanks.

**Comment by zatsme** [comment]
Yes, whilst I'm not bothered about security so much and am happy how things are, you really should allow the user to have choice here.  A choice to automatically download and install, or to just tell the user there's a new version, or not to check at all.

**Comment by Wend1go** [comment]
Small addition:
https://gdevelop-app.com/ doesn't have a privacy notice according to GDPR.
https://fr.wikipedia.org/wiki/R%C3%A8glement_g%C3%A9n%C3%A9ral_sur_la_protection_des_donn%C3%A9es

**Comment by 4ian** [comment]
I've added a card for this in Trello: https://trello.com/c/zRcuG7BP/119-add-an-option-to-disable-automatic-updates-of-the-downloaded-editor

Will see what I can do

**Comment by 4ian** [comment]
I'll be working on this this week: https://trello.com/c/zRcuG7BP/119-add-an-option-to-disable-automatic-updates-of-the-downloaded-editor-2-to-4-days

**Comment by 4ian** [comment]
I've added a preference to disable this in beta 48, closing this now. There is still a notification but I think it's important as in 99% of cases users should be aware that there is a new version and upgrade :)

**Comment by 4ian** [comment]
No thanks for something that I spend a few evenings to add and roll out in a new version for you guys? 😉
Or should I have spent my time on something else? 🤔Let me know!

**Comment by zatsme** [comment]
I wasn't bothered by the auto updates, but it could have put many potential users off and if some malware was automatically distributed by this update gdevelop's reputation could have been harmed.  So for that reason this could have been your most important work for some time, although I'm personally looking forward to tiled maps and visual hitbox editing. 👍🏻

**Comment by blurymind** [comment]
Thank you @4ian :)

**Comment by Wend1go** [comment]
Yeah thanks for implementing it. :+1: 
Although I'm pretty sure auto updates were broken on Linux since I always installed new versions manually.

**Comment by 4ian** [comment]
Auto updates are not implemented on Linux indeed (it's not part of the auto updater provided by electron-builder) - something should be done to have at least the user be notified when a new version is available. Maybe something in electron-builder to be found about this too!

**Comment by ddabrahim** [comment]
Sorry, I was completely forget about this, I wasn't expect you to do it right away. All the work you do is highly appreciated. Thanks a lot for implementing this so fast 👍
